js/blocks/ExtrasBlocks.js

```diff
@@ -395,28 +395,28 @@ function setupExtrasBlocks() {
             if (name === _("note")) {
                 switch (blockArgs.length) {
                     case 1:
-                        var p = "sol";
-                        var o = 4;
-                        var v = 4;
+                        let p = "sol";
+                        let o = 4;
+                        let v = 4;
                         break;
                     case 2:
-                        var p = blockArgs[1];
-                        var o = 4;
-                        var v = 4;
+                        p = blockArgs[1];
```

Comment:
The problem here is that the variables are only defined in case 1, not any of the other cases.

Reply:
@walterbender If I declare the variables for every case,that throws error "identifier p declared  again" and the music blocks is not running. Actually, it  is obvious, because let is used for block scoping, so in the switch block, I have to declare it only once.

Reply:
Why not declare them once before the switch?